tests/test_file_to_json.py

```diff
@@ -0,0 +1,35 @@
+
+import os
+import sys
+sys.path.append(os.path.abspath(''))
+from src.file_to_json import file_to_json  # noqa: E402
+
+
+def check_output(actual_path, expected_path):
+    with open(actual_path) as actual_file:
+        with open(expected_path) as expected_file:
+            assert actual_file.read() == expected_file.read()
+
+
+def clean_output(filepath):
+    if os.path.exists(filepath):
+        os.remove(filepath)
+
+
+def test_file_to_json():
+    try:
+        file_to_json('tests/resources', 'tests/resources', blacklist=['Key'])
+        check_output('tests/resources/Countytest_food_insecurity_xls.json',
+                     'tests/resources/test_food_insecurity_expected.json')
+        check_output('tests/resources/Countytest_food_insecurity_xlsx.json',
+                     'tests/resources/test_food_insecurity_expected.json')
+        check_output('tests/resources/test_food_insecurity_csv.json',
+                     'tests/resources/test_food_insecurity_expected.json')
+    finally:
+        clean_output('tests/resources/Countytest_food_insecurity_xls.json')
+        clean_output('tests/resources/Countytest_food_insecurity_xlsx.json')
+        clean_output('tests/resources/test_food_insecurity_csv.json')
```

Comment:
What's the intention of deleting the test output? Can't we just leave it?

Reply:
The current `.gitignore` rules would include the JSON output files. That said, it looks like I'm going to be manually carving out exceptions per Michael's comment above; in that case, I'll specifically call out these files for ignore.

Reply:
I'm backing out my changes to the .gitignore instead. Would you still prefer we keep the test files around?

Reply:
Yeah I'd like to keep them after local tests in case we want to inspect the output after the fact. How about moving the output to a new tests/output/ folder that we can exclude everything under so we don't have to maintain a specific list of files?

Reply:
Works for me.